### PR TITLE
Fb1288

### PR DESCRIPTION
**Pull request by spiresheep**

### CODE REVIEW

File: Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx [hunk 1]
```diff
@@ -1,147 +1,260 @@
...
+                  className={css((this.state.applicabilityStyle as any)[state])}
+                  style={EntitlementRow.STYLE.box.expandableTable}>
+                <HLine color='#E6E6E6'/>
+                <div style={EntitlementRow.STYLE.box.header}>
+                  <div style={EntitlementRow.STYLE.text.nameWhenExpandedTable}>
+                    Applicability
+                  </div>
+                  <div style={EntitlementRow.STYLE.box.headerFiller}/>
+                  <div style={{...amountColor,
+                      ...tableHeaderAmmountVisibility}}>
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: Typo on amount.

File: Applications/WebPortal/WebApp/library/source/components/drop_down_button.tsx [hunk 2]
```diff
@@ -151,15 +151,17 @@ export class DropDownButton extends React.Component<Properties, State> {
       animationName: DropDownButton.OPEN_AND_FADEOUT
     },
     spinClose: {
-      animationName: DropDownButton.CLOSE_AND_FADEOUT
+      animationName: DropDownButton.CLOSE_AND_FADEOUT,
+      animationDuration: '200ms'
```
[the remark this answers is not in the document]
Comment (spiresheep) [reply Applications/WebPortal/WebApp/library/source/components/drop_down_button.tsx]: Nothing, it got changed and then changed back.

File: Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx [hunk 3]
```diff
@@ -1,147 +1,272 @@
...
 import {EntitlementsPageSizing} from './entitlements_page';
+import {EntitlementTable} from './entitlement_table';
 
 interface Properties {
-  entitlementEntry?: Nexus.EntitlementDatabase.Entry;
+
+  /** The entitlement to be displayed. */
+  entitlementEntry: Nexus.EntitlementDatabase.Entry;
+
+  /** The currency in which the entitlment is delt in. */
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: Typo on entitlement. Also change delt to priced.

File: Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx [hunk 4]
```diff
@@ -1,147 +1,272 @@
...
+  marketDatabase: Nexus.MarketDatabase;
 }
 
 interface State {
-  isOpen: boolean;
+  isExpanded: boolean;
+  applicabilityStyle: any;
 }
-
+/** Displays a Entitlement Row. It displays a name, a ammount and
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: Typo on "a ammount"

File: Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx [hunk 5]
```diff
@@ -1,147 +1,272 @@
...
       }
     })();
     const amount = `${this.props.currencyEntry.sign}${
       this.props.entitlementEntry.price.toString()} ${
       this.props.currencyEntry.code}`;
     const amountColor = (() => {
       if(this.props.isActive) {
-        return EntitlementRow.STYLE.greenCheckMark;
+        if(this.state.isExpanded) {
+          return EntitlementRow.STYLE.text.activeAmmountWhenExpandedTable;
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: Typo on amount

File: Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx [hunk 6]
```diff
@@ -1,147 +1,272 @@
...
+        } else {
+          return EntitlementRow.STYLE.text.activeAmmount;
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlement_row.tsx]: Typo on amount.

**Pull request merged**